src/app/community/create/create.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-community-create',
+  templateUrl: './create.component.html',
+  styleUrls: ['./create.component.scss']
+})
+export class CreateComponent implements OnInit {
+  nameText = '';
```

Comment:
You can define these as `nameText: string;` etc...

Reply:
Fixed~ Thanks!